gnomad_constraint/constraint/constraint_pipeline.py

```diff
@@ -58,8 +102,21 @@ def main(args):
     parser.add_argument(
         "--overwrite", help="Whether to overwrite output files", action="store_true"
     )
```

Comment:
You left a suggestion here: Maybe add in a test option that filters the exome ht and the context ht to only a few partitions using `._filter_partitions(range(10))`
Does `_filter_partitions` already exist? I couldn't find it in gnomad_method or hail documentation.